### DISCUSSION TITLE
til-site/posts/doing-is-normally-distributed-learning-is-log-normal/

### DISCUSSION THREAD

**Discussion by giscus** [hiAndrewQuinn/til#14] [Announcements]
# til-site/posts/doing-is-normally-distributed-learning-is-log-normal/

There are few things I think about more than the essays on gwern.net, and there are few with as satisfying a theoretical payout to contemplate in my orb as his essay on “leaky pipelines”, aka log-normal distributions.
The skulk: Say you’re working on a Laravel web app. You’re about 90% sure you know how to start the app. You’re 80% sure you know how to handle the infra you’ll need to get it online.

https://hiandrewquinn.github.io/til-site/posts/doing-is-normally-distributed-learning-is-log-normal/

**Comment by huttj** [comment]
> but doing one estimated week-long project in half the time, and another estimated week-long project in twice the time, still leaves you in the red for your time budget.

Love this simple but clear insight.

**Comment by sfeeser** [comment]
Reconsider that first paragraph with some new numbers: 

Imagine you're working on a Laravel web app. You’re about 90% confident you know how to start the app. You’re 80% confident you can handle the infrastructure needed to get it online. And you’re 70% confident you can secure your first customer. What are your chances of successfully going from zero to your first customer? 0.9 * 0.8 * 0.7 = a little over 0.5. That’s a lot less encouraging than any of the previous numbers, if you follow this multi-step modeling approach.

Now, assume that AI is used, which increases your learning speed by 10x. This improvement significantly boosts your confidence in each step. Let’s say your confidence in each step increases by 10% due to AI assistance. With these improvements, your confidence levels become 99% for starting the app, 88% for handling the infrastructure, and 77% for securing your first customer. The calculation changes to: 0.99 * 0.88 * 0.77 = approximately 0.67. This means your chances of successfully going from zero to your first customer are now 67%, which is significantly more encouraging.  Some work needs to be done to map AI to learning speed that that this shot from the hip.
